### PR TITLE
Toml cleanup

### PR DESCRIPTION
**Pull request by brown170**

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -1,8 +1,8 @@
-numpy
```
Comment (julia-sprenger) [inline requirements.txt:1]: I think the toml would be a better file to keep track of the dependencies. Pip should also be able to install dependencies from a toml file. Is there a reason to keep these in a dedicated file?

Comment (brown170) [reply requirements.txt:1]: I agree, but I did some tests on my computer.  If I put the dependencies in the toml file, then pip did not process them.  I do not know if this is a Dave-computer-specific behavior or a general feature of pip.  I could not locate any information about pip's handling of toml files, but I also did not google very hard.

Comment (brown170) [reply requirements.txt:1]: specifically, I was pip installing the requirements/dependencies into a virtual environment

Comment (brown170) [reply requirements.txt:1]: Interesting that I am using pip version 24.1.1 and pip instructions (24 & newer) recommend dependencies in the toml file.

Comment (brown170) [reply requirements.txt:1]: I think the solution here is to make a tracker, merge this pull request and keep an eye on the situation.  The toml file is a much cleaner solution, but inconsistent behavior across platforms is problematic.

**Pull request merged**